tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -14,9 +21,40 @@ Verify OpenVino Is Available In RHODS Dashboard Explore Page
   Verify Service Is Available In The Explore Page    OpenVINO
   Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
 
-* Keywords ***
+Verify Openvino Operator Can Be Installed Using OpenShift Console
+   [Tags]  ODS-675   ODS-702  Sanity
+   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster
+   ...               and Check and Launch AIKIT notebook image from RHODS dashboard
+   Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
+   Create Tabname Instance For Installed Operator        ${openvino_operator_name}       Notebook    redhat-ods-applications
+   Wait Until Keyword Succeeds    900  1     Check Image Build Status   Complete     openvino-notebook
+   Verify If Openvino Service is Enabled in RHODS Dashboard
+   Verify JupyterHub Can Spawn Openvino Notebook
+   [Teardown]   Uninstall Openvino Operator
+
+*** Keywords ***
 OpenVino Suite Setup
   Set Library Search Order  SeleniumLibrary
 
 OpenVino Suite Teardown
   Close All Browsers
+
+Uninstall Openvino Operator
+    Go To  ${OCP_CONSOLE_URL}
+    Maybe Skip Tour
+    Delete Tabname Instance For Installed Operator      ${openvino_operator_name}       Notebook    redhat-ods-applications
+    Uninstall Operator       ${openvino_operator_name}
+
+Verify If Openvino Service is Enabled in RHODS Dashboard
```

Comment:
This is redundant with AiKit. I would create a single keyword or just create a new keyword named `Go To Dashboard` (if not already present) which implement Lines 49-50-51. Then I would change the TC ODS-675 to do:
```
Go to Dashboard
Verify Service Is Enabled
```
I think this is something will go through refactoring in the next weeks/months